app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -204,13 +207,21 @@ public boolean onOptionsItemSelected(final MenuItem item) {
 
     @Override
     protected void onUpdatePrimaryLanguage() {
-        updateLatestTranslationLoader();
+        updateLatestPrimaryTranslationLoader();
 
         // restart the progress listener
         stopProgressListener();
         startProgressListener();
     }
 
+    @Override
+    protected void onUpdateParallelLanguage() {
+        updateLatestParallelTranslationLoader();
+
+        stopProgressListener();
+        startProgressListener();
```

Comment:
we don't need to restart the progress listener on changing the parallel language. the progress listener only monitors the primary language download